src/figuresClasses.ts

```diff
@@ -1,11 +1,101 @@
-export interface Figure {}
+export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: 'triangle' | 'circle' | 'rectangle';
+  getArea(): number;
+}
+// Triangle Class
+export class Triangle implements Figure {
+  color: 'red' | 'green' | 'blue';
```

Comment:
you should declare type Color and use it everywhere

 type Color = 'red' | 'green' | 'blue';